### DISCUSSION TITLE
Compare to Rancher and Kubesphere

### DISCUSSION THREAD

**Discussion by wonderflow** [karmada-io/karmada#289] [Q&A]
Hi, I wonder what's the difference between karmada and [Rancher](https://github.com/rancher/rancher) and  [Kubesphere](https://github.com/kubesphere/kubesphere) ? Thanks.

**Comment by lfbear** [comment]
As a trial user for karmada, I consider that Rancher and Kubesphere are platforms I can easily manage multi-clusters by web UI, those are fit for operators, or some engineers. However, end-users need not care how those manage multi-clusters, what the pattern. Because those questions should be cared by those platforms and some of the questions in those platforms may be similar, but there will be many different ways created to solve. So that must need an underlying capability, common and native way. I think karmada will be that. Some platforms may use karmada as a core for multi-clusters management, I think.

**Comment by nivalume** [comment]
I agree with @lfbear  about karmadaas a core for multi-clusters management. In fact we currently do, we use Kubesphere as the PAAS platform and karmada as core for multi-cluster management, schedule.
